app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+end
```

Comment:
- [ ] In this file as well, it is required that the Comment model has a method that updates the comments counter for a post.

**Also, you could make use of the after_save here as well for every update that happens**